code/datums/atmosphere/planetary.dm

```diff
@@ -19,7 +19,7 @@
 	)
 	restricted_chance = 30
 
-	minimum_pressure = HAZARD_LOW_PRESSURE + 10
+	minimum_pressure = HAZARD_LOW_PRESSURE + 11
```

Comment:
This is a hacky solution since you are changing the entire planets gas pressure for something very minor.  The better fix would be to change the wings themselves to adjust to the lowest generated pressure.